util/build_configs/cray-internal/setenv-xc-x86_64.bash

```diff
@@ -353,10 +352,6 @@ else
         # load target PrgEnv with compiler version
         load_module $target_prgenv
         load_module_version $target_compiler $target_version
-
-        # pin to versions of mpich/libsci that work with gen_version_gcc
-        load_module_version cray-mpich 7.7.13
-        load_module_version cray-libsci 19.06.1
```

Comment:
These are still needed with the older gcc.

Reply:
:(